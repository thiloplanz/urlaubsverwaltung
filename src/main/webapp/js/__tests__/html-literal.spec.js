import html from '../html-literal'

describe('html-literal', () => {
  it('returns processed string', () => {
    const title = "awesome stuff!";
    const actual = html`<h1>${title}</h1>`;
    expect(actual).toMatchSnapshot();
  });

  it('returns processed string with array substitutions', () => {
    const personList = [
      'Bruce',
      'Clark'
    ];
    const actual = html`<ul>${personList.map(person => `<li>${person}</li>`)}</ul>`;
    expect(actual).toMatchSnapshot();
  });
});
