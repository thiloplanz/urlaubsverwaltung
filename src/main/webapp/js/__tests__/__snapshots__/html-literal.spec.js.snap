// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`html-literal returns processed string 1`] = `"<h1>awesome stuff!</h1>"`;

exports[`html-literal returns processed string with array substitutions 1`] = `"<ul><li>Bruce</li><li>Clark</li></ul>"`;
